transformer/src/main/scala/com/snowplowanalytics/snowflake/transformer/TransformerJob.scala

```diff
@@ -52,13 +52,33 @@ object TransformerJob {
     sc.register(keysAggregator)
 
     val events = sc
-      .textFile(jobConfig.input)
-      .map { e => Transformer.jsonify(e) }
+      .wholeTextFiles(jobConfig.input)
+      .flatMap(rdd => {
+        val fileName = rdd._1
+        val fileContents = rdd._2
```

Comment:
Is it possible to do a `flatMap { case (fileName, fileContents) =>` to avoid these two lines, as well as the additional pairs of brackets?

Reply:
Yes, very cool, thanks!